src/Core/Components/NavMenu/FluentNavMenu.razor

```diff
@@ -1,12 +1,13 @@
 ﻿@namespace Microsoft.Fast.Components.FluentUI
 @inherits FluentComponentBase
 
-<div id="@Id" @ref="@Element" @attributes="AdditionalAttributes" class="@ClassValue" style="@StyleValue" aria-label="@(Collapsible ? null : @Title)">
+<div id="@Id" @ref="@Element" @attributes="AdditionalAttributes" class="@ClassValue" style="@StyleValue" aria-label="@(Collapsible ? null : @Title)" role="menu" aria-expanded="@Expanded">
```

Comment:
Yes because when `Collapsible=false` the `div´ of Line 9 is not included in the HTML content. But the Accessiblity rule required a tag with role=menu and a aria-expanded